main/src/library/Iterable.flix

```diff
@@ -17,17 +17,26 @@
 ///
 /// A trait for immutable data structures that can be iterated.
 ///
-pub trait Iterable[t: Type -> Type] {
+pub trait Iterable[t] {
+    ///
+    /// The element type of the Iterable.
+    ///
+    type Elm[t]: Type
```

Comment:
I think its good to put kind annotations on both Elm and Aef 👍

Reply:
Thanks